router.go

```diff
@@ -10,8 +11,27 @@ import (
 	"k8s.io/kubernetes/pkg/util"
 )
 
+// Do we want to run nginx without kubernetes driving configuration?
+var kubeless bool
+
+func init() {
+	const (
```

Comment:
no technical harm in doing it, recommend we do this a different way and put the constants package-global and the flag handling in the main.

```go
func main() {
  kubeless := flag.Bool("kubeless", defaultKubeless, kubelessUsage)
  if *kubeless {
    ...
  }
  ...
```